base/src/main/java/uk/ac/ebi/atlas/model/card/CardIconType.java

```diff
@@ -1,5 +1,5 @@
 package uk.ac.ebi.atlas.model.card;
 
 public enum CardIconType {
-    SPECIES, IMAGE
+    SPECIES, IMAGE, ATLASSPECIES
```

Comment:
atlas species card has differential and baseline button

Reply:
This enum does not denote the card type, but the type of icon used in the card. I.e. a `SPECIES` icon type is represented by the EBI species icons, and a `IMAGE` icon type will make use of static image assets such as `.png` files. How is the `ATLASSPECIES` icon type different?

Reply:
Because we need to render different type of cards when we received the card type from backend. `ATLASSPECIES` specified that it needs a species card with B and D links, which is different from the rendering of `SPECIES` cards.